app/controllers/frontend_log_controller.rb

```diff
@@ -1,4 +1,6 @@
 class FrontendLogController < ApplicationController
+  include Idv::AnalyticsConcern
```

Comment:
As I'm starting to have to include this concern in controllers which aren't strictly related to IdV, I wonder if we'd just be better off to include the decorator in the base `Analytics` class, which would help avoid having to litter it throughout, while still keeping it separate enough from the rest of the default analytics behaviors.

Reply:
> As I'm starting to have to include this concern in controllers which aren't strictly related to IdV, I wonder if we'd just be better off to include the decorator in the base `Analytics` class, which would help avoid having to litter it throughout, while still keeping it separate enough from the rest of the default analytics behaviors.

Updated in 11fc94c